docs/pix2struct-chyrons/v1.0/metadata.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "Pix2struct Chyrons",
+  "description": "extracts text from input timeframes based on chyron queries using the pix2struct Doc-VQA model",
+  "app_version": "v1.0",
+  "mmif_version": "1.0.0",
+  "analyzer_version": "1",
+  "app_license": "MIT",
+  "analyzer_license": "apache-2.0",
+  "identifier": "http://apps.clams.ai/pix2struct-chyrons/v1.0",
+  "url": "https://github.com/clamsproject/app-pix2struct-docvqa-wrapper",
+  "input": [
+    {
+      "@type": "http://mmif.clams.ai/vocabulary/VideoDocument/v1",
+      "required": true
+    }
+  ],
```

Comment:
From https://github.com/clamsproject/app-pix2struct-chyrons/blob/cd959fb8b7954769de590924670cdb53e9c475c0/app.py#L60-L75 , the app seems to require `TimeFrame` in the input MMIF but the type is not specified in the metadata.